.gitignore

```diff
@@ -13,4 +13,5 @@ Localize/out/
 /packages/playground/resources/i18n/*
 /packages/react/resources/i18n/*
 /packages/service/resources/i18n/*
-/util/bux/resources/i18n/json
+web/dev-server/resources/i18n/*
```

Comment:
It looks like web/resources/i18n/* needs to be added .gitignore. The .json file is currently checked in to source control.

Reply:
I don't see a web/resources directory, either generated during a build step or in source control: https://github.com/Azure/BatchExplorer/tree/dpwatrous/fake-localizer/web. 

(The web translations are stored in `web/dev-server/resources/i18n/*` since the server can only access resources inside the dev-server folder)

Reply:
Pretty sure this was my mistake due to switching branches and there's no issue here - sorry about that!